### ISSUE TITLE
[Bug] When used with laravel octane, css is not loaded.

### ISSUE THREAD

**Issue by paulzakk** [Laravel-Backpack/basset#102]
I am using php 8.1 version and laravel 10, backpack 6.1.15 version.
Laravel is configured with octane (swoole).
When I log in after completing the configuration, it seems that the css file is not called. Could this be a bug?

**Comment by alruddick** [comment]
When binding the `BassetManager` in `BassetServiceProvider`, using a Singleton w/ Octane can cause problems on subsequent requests. Changing this to a normal binding resolves the issue, however, I'm not 100% sure why a singleton was used in the first place. Someone else might be able to shed some light on that

```
    /**
     * Register any package services.
     *
     * @return void
     */
    public function register(): void
    {
        // Register the service the package provides.
        $this->app->bind('basset', fn () => new BassetManager());

        // Merge the configuration file.
        $this->mergeConfigFrom(__DIR__.'/config/backpack/basset.php', 'backpack.basset');

        // Register blade directives
        $this->registerBladeDirectives();
    }
```

**Comment by promatik** [comment]
Hi everyone!
Indeed Basset was being loaded as a `singleton` instead of `scoped`, so that was leading to files being marked as loaded forever 🤷‍♂️
That should be fixed with #105, tagged on https://github.com/Laravel-Backpack/basset/releases/tag/1.2.2 🙌
